src/scripts/background.js

```diff
@@ -18,6 +19,22 @@ const shouldTriggerNotification = (state, seconds) => {
   );
 };
 
+function randomInt (min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+}
+
+function jitter () {
+  // Random amount between 0-5 seconds
+  return randomInt(0, 5);
```

Comment:
I think, given the range of users we have on different types of connection, it's already true there's about 5 seconds of jitter (though most users would probably take 0-3 seconds to connect, not 5 seconds).

What do you think about changing this range from 0..5 to 0..30, or even 0..60? IMO 5s jitter is very little

Reply:
I'd bump either the base amount or jitter, leaning towards jitter